From 6a2054424bc670339117024a9800fe556d538438 Mon Sep 17 00:00:00 2001
From: axis <qt-info@nokia.com>
Date: Mon, 7 Mar 2011 16:14:39 +0100
Subject: [PATCH] Removed modular references after support for the flag was removed.

Support was removed because the Qt Modularization project decided we
don't need it. It's better to base decisions on the available
information, as demonstrated in the patch by checking
QT.phonon.includes.

In addition, remove the reference to uitools. It has been moved to
QtBase now, and therefore is always available.
---
 Source/WebKit.pri |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/Source/WebKit.pri b/Source/WebKit.pri
index 8785496..90ae6fd 100644
--- a/Source/WebKit.pri
+++ b/Source/WebKit.pri
@@ -134,9 +134,7 @@ symbian|maemo5|maemo6 {
 
 ####
 
-contains(QT_CONFIG, modular):!contains(QT_CONFIG, uitools)|disable_uitools: DEFINES *= QT_NO_UITOOLS
-
-!contains(QT_CONFIG, modular) {
+isEmpty(QT.phonon.includes) {
     QT.phonon.includes = $$QMAKE_INCDIR_QT/phonon
     QT.phonon.libs = $$QMAKE_LIBDIR_QT
 }
-- 
1.7.2.2

